subsys/bluetooth/controller/cs_antenna_switch.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ #include <stdint.h>
+ #include "cs_antenna_switch.h"
+
+#include <hal/nrf_gpio.h>
+
+#define DEFAULT_CS_ANTENNA_GPIO_PORT NRF_P1
+#define DEFAULT_CS_ANTENNA_BASE_PIN (11)
+#define DEFAULT_CS_ANTENNA_PIN_MASK (0xF << DEFAULT_CS_ANTENNA_BASE_PIN)
```

Comment:
there should be no default in a C file, it should come from dts

Reply:
Hi, yes, I agree with you that, eventually, this will come from DTS. That is actually what we have planned in our team.

However in this first step we're keeping our existing hard-coded controller code (just moving it from the compiled library to these open source files) so that we can make this API change, which is a multi-step process.

We have not yet settled on what exactly the DTS configuration or its docs will look like and may need to collect more customer feedback before implementing it, so there is no point delaying this PR for that.